code/modules/mob/dead/observer/spells.dm

```diff
@@ -55,4 +55,5 @@ var/global/list/boo_phrases_silicon=list(
 /spell/aoe_turf/boo/cast(list/targets)
 	for(var/turf/T in targets)
 		for(var/atom/A in T.contents)
-			A.spook()
+			if(A.can_spook())
```

Comment:
If you're going to put this restriction in, it might be working adding can_spook to the casting checker proc so people don't waste their Boo!